rascal-textmate-core/src/main/rascal/lang/textmate/conversiontests/Pico.rsc

```diff
@@ -14,8 +14,8 @@ Grammar rsc = preprocess(grammar(#Program));
 
 list[ConversionUnit] units = [
```

Comment:
These tests are difficult to read. Some comments about what 'units' represent would be helpful. It looks like an encoding of some Rascal syntax rules. Is it not possible to use the actual rules in Rascal and parse them?

This is even more of an issue with the Rascal tests.

Reply:
Yes, I agree these tests are hard to read. I'll think a bit more about if there is a better way to achieve the same aim.

Units are documented in the [module](https://github.com/SWAT-engineering/rascal-textmate/blob/main/rascal-textmate-core/src/main/rascal/lang/textmate/ConversionUnit.rsc) that defines the type. It's basically a `prod` (verbatim from the grammar) with some metadata (computed during the conversion).